src/main/scala/uk/gov/hmrc/ct/ct600/v3/calculations/B325Calculator.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.ct600.v3.calculations
+
+import uk.gov.hmrc.ct.box.CtTypeConverters
+import uk.gov.hmrc.ct.ct600.v3._
+
+trait B325Calculator extends CtTypeConverters {
+
+  def calculateB325(b335: B335, b385: B385, b315: B315, b330: B330, b380: B380): B325 = {
+
+    (b330, b380) match {
+
+      case singleFinancialYear if singleFinancialYear._1.isPositive && singleFinancialYear._2.noValue &&  b335.value <= b315.value => B325(Some(b335.value))
+      case twoFinancialYears if twoFinancialYears._1.isPositive && twoFinancialYears._2.hasValue && (b335.value + b385.value) <= b315.value  => B325(Some(b335.value + b385.value))
+      case _ => B325(Some(0))
```

Comment:
Why not None?

Reply:
hmm... about the Option. I'm not sure if I said properly.
It depends on how those boxes are used further. If it is always loaded then Option might be wrong...

Reply:
I've checked file CT-specDoc-v1-9.pdf
both boxes here have cardinality 0..1

Reply:
We need to know when this 0 means we should type None
I think when a person is not eligible

Reply:
Alright I will change that to None as I thought we can return 0 if its exceed the value but anyways i will change it for you.